From 099d1c1f3376673e9a3c5747c87c8c756c883ce6 Mon Sep 17 00:00:00 2001
From: Andy Ning <andy.ning@windriver.com>
Date: Thu, 22 Mar 2018 11:45:26 -0400
Subject: [PATCH 1/1] CGTS-9265: patch to harden server and client config

Replace the hardcoded sshd_config and ssh_config files with patches
to openssh.

Signed-off-by: Andy Ning <andy.ning@windriver.com>
---
 SPECS/openssh.spec | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/SPECS/openssh.spec b/SPECS/openssh.spec
index 7ee5bd8..7c464a2 100644
--- a/SPECS/openssh.spec
+++ b/SPECS/openssh.spec
@@ -240,6 +240,9 @@ Patch955: openssh-7.4p1-sandbox-ibmca.patch
 # Back to UseDNS=yes by default (#1478175)
 Patch956: openssh-7.4p1-usedns-yes.patch
 
+# WRS: harden server and client config
+Patch1000: harden-server-and-client-config.patch
+
 License: BSD
 Group: Applications/Internet
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
@@ -494,6 +497,9 @@ popd
 
 %patch100 -p1 -b .coverity
 
+# WRS
+%patch1000 -p1 -b .harden
+
 %if 0
 # Nothing here yet
 %endif
-- 
1.8.3.1

